downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -218,13 +218,23 @@ class Git : VersionControlSystem(), CommandLineTool {
             logger.info { "Falling back to fetch everything including tags." }
 
             git.fetch().setUnshallow(true).setTagOpt(TagOpt.FETCH_TAGS).call()
+        }.recoverCatching {
+            it.showStackTrace()
+
+            logger.info { "Could not fetch everything using JGit: ${it.collectMessages()}" }
+            logger.info { "Could not fetch everything using JGit. Falling back to Git CLI." }
```

Comment:
I believe only one of these / a combination of both would be enough.

Reply:
And maybe explicitly say that this partly reverts d899156a73fb8cbe82f856c6b01e12f599487f57.

Reply:
> I believe only one of these / a combination of both would be enough.

I accidentally duplicated, added the failure message. Removed this. Now the logs are following the previous fallback logs.